DatadogSessionReplay/Sources/Recorder/Recorder.swift

```diff
@@ -116,6 +120,12 @@ public class Recorder: Recording {
     /// Initiates the capture of a next record.
     /// **Note**: This is called on the main thread.
     func captureNextRecord(_ recorderContext: Context) {
+        let methodCalledTrace = telemetry.startMethodCalled(
+            operationName: MethodCallConstants.captureRecordOperationName,
+            callerClass: MethodCallConstants.className,
+            samplingRate: methodCallTelemetrySamplingRate // Effectively 3% * 15% = 0.45% of calls
```

Comment:
I can definitely be lower. Even 5% seems enough.

As discussed - maybe all telemetries should be sampled on the call side instead. This gives more control for future metrics, and removes the need to remember the math of `20% * 15% * Y% = Desired%`.

Right now whatever % we choose for the call side sampling will have to unnecessarily travel through the message bus to be rejected by top level sampling.

Reply:
https://datadoghq.atlassian.net/browse/RUM-3572

Reply:
my bigger concern about such telemetries are about turning them off.

Once this code is out in the app, we will continue getting this telemetry even though we no longer need them because the concern has been addressed or this is way too much of data or etc.

This brings the topic of remote configs but we are not there yet.